[Bug]: error with installing dependencies for vue

### Describe the Bug

Hi!

There was an error while installing the dependencies for `@storefront-ui/vue.` This package cannot be installed using npm and pnpm, but it was successfully installed when I used Yarn.

1. here is error by pnpm
```
pnpm add -D @nuxtjs/tailwindcss @storefront-ui/vue   
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@storefront-ui/tw-plugin-peer-next@workspace:*" is in the dependencies but no package named "@storefront-ui/tw-plugin-peer-next" is present in the workspace
```
2. and error by npm
```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

npm ERR! A complete log of this run can be found in:
```
Here is link to stackblitz: https://stackblitz.com/edit/nuxt-starter-xnjcby?file=README.md

### Expected behavior

These packages should be installed.

### Steps to reproduce

_No response_

### SFUI version

@storefront-ui/vue@2.3.0

### Framework

Nuxt

### Node version

v19.0.0

### Browser

does not apply

### OS

macOS

### Relevant log output

_No response_

### Able to fix / change the documentation?

- [ ] Yes
- [ ] No

### Code of Conduct

- [ ] I agree to follow this project's Code of Conduct

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,9 @@
+# 2.3.1 (2023-06-08)
+
+## Bug fixes
+
+- fix package cross-dependencies that caused released package to fail
+
 # 2.3.0 (2023-06-07)
 
 ## Overview

--- a/packages/sfui/frameworks/nuxt/package.json
+++ b/packages/sfui/frameworks/nuxt/package.json
@@ -36,7 +36,7 @@
   "dependencies": {
     "@nuxt/kit": "^3.3.2",
     "@nuxtjs/tailwindcss": "^6.6.4",
-    "@storefront-ui/vue": "latest"
+    "@storefront-ui/vue": "workspace:*"
   },
   "devDependencies": {
     "@nuxt/module-builder": "^0.2.1",

